content/chat/moderation/byo/index.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Bring Your Own Moderation
+meta_description: "Detect and remove unwanted content in a Chat Room using a custom provider"
+product: chat
+---
+
+There may be situations where you have trained your own model, or you want to apply proprietary logic using your own infrastructure, whilst performing moderation.
+
+Ably provides simple APIs to allow your moderation logic to prevent harmful content from being present in your chat room.
```

Comment:
A lot of the `allows` here should technically be `enables`.

Reply:
Done!